common/crypto_utils/setup.py

```diff
@@ -58,59 +42,31 @@
 
 crypto_include_dirs = [
     os.path.join(tcf_root_dir, 'common/cpp'),
```

Comment:
avalon_crypto_utils uses Pycrytodomex and is not dependent on cpp.

Reply:
Done